src/ProcessManager/ChromeManager.php

```diff
@@ -30,13 +30,28 @@ final class ChromeManager implements BrowserManagerInterface
     private $arguments;
     private $options;
 
+    private $extensions = [];
+
     public function __construct(?string $chromeDriverBinary = null, ?array $arguments = null, array $options = [])
     {
         $this->options = array_merge($this->getDefaultOptions(), $options);
         $this->process = new Process([$chromeDriverBinary ?: $this->findChromeDriverBinary(), '--port='.$this->options['port']], null, null, null, null);
         $this->arguments = $arguments ?? $this->getDefaultArguments();
     }
 
+    /**
+     * @param string[] $extensions Paths to packed Chrome extensions
+     */
+    public function setExtensions(array $extensions)
+    {
+        foreach ($extensions as $path) {
+            if (!file_exists($path)) {
+                throw new \InvalidArgumentException(\sprintf('The Chrome extension "%s" does not exist.', $path));
```

Comment:
Nice changes to the original PR!

Can there be an additional test verifying that this exception is indeed thrown?

Doing so ensures that future changes can't inadvertently break this logic.

Reply:
I added new test case. Thanks for reviewing!